acl_setup.php

```diff
@@ -174,6 +181,18 @@
  $gacl->add_object('patients', 'Lab Results (write,addonly optional)','lab', 10, 0, 'ACO');
      // xl('Lab Results (write,addonly optional)')
 
+
+ $gacl->add_object('groups', 'View/Add/Update groups','gadd', 10, 0, 'ACO');
+ // xl('View/Add/Update groups groups')
```

Comment:
there is an extra 'groups' in the above comment (btw, this is what goes to the translation pipeline)